### PR TITLE
Update wheel versions in documentation to 0.8.0.dev0 and 0.7.0.

### PR DESCRIPTION
**Pull request by devin-petersohn**
## What do these changes do?

Performance integration testing is failing because of a version mismatch.

## Linter

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

**Comment by devin-petersohn** [comment]
This will likely fail the performance integration testing until #4845 is merged.

**Comment by ericl** [comment]
Is this related to the normal jenkins build breaking with version mismatch on 0.8.0 vs 0.7.0?

**Review by robertnishihara — COMMENTED** [review]
@devin-petersohn we need to change the wheel versions in a lot of other places as well.

Any instance of `0.7.0.dev3` or `0.6.6` in the codebase should probably be replaced.

**Comment by robertnishihara** [comment]
@ericl that's a very weird error. Do you have a link to a jenkins log that does that?

**Comment by ericl** [comment]
#4849 is failing with errors like
```
/opt/conda/lib/python2.7/site-packages/ray/worker.py", line 1762, in connect
    ray.services.check_version_info(worker.redis_client)
  File "/opt/conda/lib/python2.7/site-packages/ray/services.py", line 496, in check_version_info
    raise Exception(error_message)
Exception: Version mismatch: The cluster was started with:
    Ray: 0.8.0.dev0
    Python: 2.7.15
    Pyarrow: 0.12.0.RAY
This process on node 172.17.0.2 was started with:
    Ray: 0.7.0.dev3
    Python: 2.7.15
    Pyarrow: 0.12.0.RAY
```

**Comment by robertnishihara** [comment]
@ericl ok, yeah I think that is related to this PR. Note that there are two Jenkins tests (Ray-PRB and Ray-Perf-Integration-PRB). The first one is the main one and the second one is the performance testing one, which is failing.

**Comment by ericl** [comment]
Yes, both are failing. By the way, it's quite annoying that the Ray perf
builder overwrites the results of the normal test one. Could we turn that
off?

**Comment by devin-petersohn** [comment]
@ericl I'm not sure if we can turn that off, we want true failures to be flagged as such anyway, otherwise it's not useful to even run it. Right now the failures are an issue because of the version bump.

@robertnishihara I pushed some other update to the version number. This was all I found, if you know of other locations please let me know.

**Comment by robertnishihara** [comment]
@devin-petersohn we also need to change the PyPI badge on the main readme.

**Comment by ericl** [comment]
The problem is it overwrites the result of the normal integration build, so it can look like it's passing, but is actually failed the integration prb.

How about we gate it so it only runs on core changes?

**Comment by robertnishihara** [comment]
@ericl I actually tried to turn it off a little while back but couldn’t find a config option for that in Jenkins.

File: doc/source/installation.rst [hunk 1]
```diff
@@ -33,14 +33,14 @@ Here are links to the latest wheels (which are built off of master). To install
...
 
-.. _`Linux Python 3.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp37-cp37m-manylinux1_x86_64.whl
-.. _`Linux Python 3.6`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp36-cp36m-manylinux1_x86_64.whl
-.. _`Linux Python 3.5`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp35-cp35m-manylinux1_x86_64.whl
-.. _`Linux Python 2.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp27-cp27mu-manylinux1_x86_64.whl
-.. _`MacOS Python 3.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp37-cp37m-macosx_10_6_intel.whl
-.. _`MacOS Python 3.6`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp36-cp36m-macosx_10_6_intel.whl
-.. _`MacOS Python 3.5`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp35-cp35m-macosx_10_6_intel.whl
-.. _`MacOS Python 2.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.7.0.dev2-cp27-cp27m-macosx_10_6_intel.whl
+.. _`Linux Python 3.7`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev0-cp37-cp37m-manylinux1_x86_64.whl
```
Comment (robertnishihara) [inline doc/source/installation.rst]: These ones are the most important to update because this is how users get the nightly wheels.

**Review by robertnishihara — COMMENTED** [review]
@devin-petersohn there were more that were not updated. I just pushed an update, please take a look.

**Review by robertnishihara — APPROVED** [review]

**Pull request merged**